src/components/registerV2/backupPassphrase.css

```diff
@@ -27,45 +28,113 @@
   width: 100%;
 
   & > h2 {
+    @mixin contentNormal bold;
+
     align-items: center;
     color: var(--color-content-light);
     display: flex;
-    font-family: var(--content-font);
-    font-size: var(--paragraph-font-size-l);
-    font-weight: var(--font-weight-semi-bold);
-    line-height: 1.47;
     margin: 0;
   }
 
-  & > p {
-    color: var(--color-content);
-    font-family: var(--heading-font);
-    font-size: var(--paragraph-font-size-s);
-    font-weight: var(--font-weight-bold);
-    letter-spacing: 0.1px;
-    margin: 15px 0;
-    text-align: left;
-    word-spacing: 8px;
+  & > .downloadLisk {
+    display: flex;
+
+    & > p {
+      color: var(--color-content);
+      font-family: var(--heading-font);
+      font-size: var(--paragraph-font-size-s);
+      font-weight: var(--font-weight-bold);
+      letter-spacing: 0.1px;
+      margin: 15px 0;
+      text-align: left;
+      word-spacing: 8px;
+    }
+
+    & > img {
+      margin-right: 10px;
+    }
   }
 
   & .action {
-    color: var(--color-primary-standard);
+    @mixin headingExtraSmall;
+
+    width: 136px;
+    height: 32px;
+    border-radius: 3px;
+    border: solid 1px rgba(0, 0, 0, 0.15);
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    color: var(--color-content-light);
     cursor: pointer;
-    font-family: var(--content-font);
-    font-size: var(--paragraph-font-size-l);
-    font-weight: var(--font-weight-semi-bold);
-    line-height: 1.47;
-    transition: color var(--animation-speed-fast) linear;
-    text-decoration: underline;
+    margin-top: 15px;
 
     &.copied {
-      color: var(--color-primary-light);
       cursor: default;
       text-decoration: none;
     }
   }
 }
 
+.passphraseField {
+  width: 99px;
+  height: 48px;
+  border-radius: 3px;
+  border: solid 1px rgba(0, 0, 0, 0.08);
+  background-color: #fff;
+  margin-right: 8px;
+  margin-bottom: 15px;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+
+  & div {
+    @mixin contentLargest;
+  }
+}
+
+.hrSection {
+  @mixin contentSmallest;
+
+  display: flex;
+  flex-basis: 100%;
+  align-items: center;
+  margin: 8px 0px;
+  color: var(--color-slate-gray);
+
+  & > p {
+    margin: 20px;
+  }
+}
+
+.hrSection::before,
+.hrSection::after {
+  content: "";
+  flex-grow: 1;
+  background: rgba(0, 0, 0, 0.35);
```

Comment:
use the blue-gray color here, it's close enough and we can still use the variables.